ci/run-bot-bulk-deactivate.yml

```diff
@@ -0,0 +1,24 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: python
+    tag: 3.8
+
+inputs:
+- name: src-repository
+
+outputs:
+- name: uaa-bot-summary
+
+run:
+  path: bash
+  args:
+  - -c
+  - |
+    cd src-repository
+    export BUILD_DATE=$(date "+%Y-%m-%d")
+    pip install --upgrade pip
+    pip install -r requirements.txt
+    python ./cli bulk-deactivate -d $DAYS_AGO -r $NUMBER_DAYS_BEFORE -t "$SUMMARY_TITLE" >> "../uaa-bot-summary/$BUILD_JOB_NAME-summary-$BUILD_DATE.json"
```

Comment:
Aha, that's how it's done (I see the changes in the pipeline file, too)!  But, I only see `$BUILD_DATE` defined - is `$BUILD_JOB_NAME` coming through just fine?  Is it because it's explicitly defined in the pipeline now?

Reply:
It's coming through via the params for the task and I just set the `BUILD_JOB_NAME` to a string like (`production-notify-deactivate`) in the pipeline.yml for each task.